File: CursoWebCompleto/exer/listaAlgoritimos/algoritimo/exer08.js
/*
8. Faça um algoritmo para ler três notas de um aluno em uma disciplina e imprimir a sua média
ponderada (as notas tem pesos respectivos de 1, 2 e 3).
 */

const grade1 = 10;
const grade2 = 10;
const grade3 = 10;

const weight1 = 1;
const weight2 = 2;
const weight3 = 3;

const average = (grade1 * weight1 + grade2 * weight2 + grade3 * weight3) / (weight1 + weight2 + weight3);

console.log(`A média ponderada é ${average}`);

File: CursoJava/Exer/Algoritimo/src/Exer08.java
/*
8. Faça um algoritmo para ler três notas de um aluno em uma disciplina e imprimir a sua média
ponderada (as notas tem pesos respectivos de 1, 2 e 3).
 */

import java.util.Scanner;

public class Exer08 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Digite a primeira nota: ");
        double nota1 = sc.nextDouble();

        System.out.println("Digite a segunda nota: ");
        double nota2 = sc.nextDouble();

        System.out.println("Digite a terceira nota: ");
        double nota3 = sc.nextDouble();

        double media = (nota1 * 1 + nota2 * 2 + nota3 * 3) / 6;

        System.out.println("A média ponderada é: " + media);

        sc.close();
    }
}
